pmlc/dialect/pxa/transforms/stencil_gemm.cc

```diff
@@ -260,12 +264,57 @@ class StencilGEMM : public StencilBase {
   }
 
   void transform(TensorAndIndexPermutation perm, ArrayRef<int64_t> tileSize) {
+    PxaGemmOp gemm;
```

Comment:
What's this for?

Reply:
I removed this stray variable declaration. Thanks for pointing it out.